vc_zoom/indico_vc_zoom/plugin.py

```diff
@@ -0,0 +1,464 @@
+                                  widget=SwitchWidget(),
+                                  description=_('Allow webinars to be created through Indico'))
+
+    mute_audio = BooleanField(_('Mute audio'),
+                              widget=SwitchWidget(),
+                              description=_('Participants will join the VC room muted by default '))
+
+    mute_host_video = BooleanField(_('Mute video (host)'),
+                                   widget=SwitchWidget(),
+                                   description=_('The host will join the VC room with video disabled'))
+
+    mute_participant_video = BooleanField(_('Mute video (participants)'),
+                                          widget=SwitchWidget(),
+                                          description=_('Participants will join the VC room with video disabled'))
+
+    join_before_host = BooleanField(_('Join Before Host'),
+                                    widget=SwitchWidget(),
+                                    description=_('Allow participants to join the meeting before the host starts the '
+                                                  'meeting. Only used for scheduled or recurring meetings.'))
+
+    waiting_room = BooleanField(_('Waiting room'),
+                                widget=SwitchWidget(),
+                                description=_('Participants may be kept in a waiting room by the host'))
+
+    zoom_phone_link = URLField(_('ZoomVoice phone number'),
+                               description=_('Link to the list of ZoomVoice phone numbers'))
+
+    creation_email_footer = TextAreaField(_('Creation email footer'), widget=CKEditorWidget(),
+                                          description=_('Footer to append to emails sent upon creation of a VC room'))
+
+    send_host_url = BooleanField(_('Send host URL'),
+                                 widget=SwitchWidget(),
+                                 description=_('Whether to send an e-mail with the Host URL to the meeting host upon '
+                                               'creation of a meeting'))
+
+
+class ZoomPlugin(VCPluginMixin, IndicoPlugin):
+    """Zoom
+
+    Zoom Plugin for Indico."""
+
+    configurable = True
+    settings_form = PluginSettingsForm
+    vc_room_form = VCRoomForm
+    vc_room_attach_form = VCRoomAttachForm
+    friendly_name = 'Zoom'
+
+    def init(self):
+        super(ZoomPlugin, self).init()
+        self.connect(signals.plugin.cli, self._extend_indico_cli)
+        self.connect(signals.event.times_changed, self._times_changed)
+        self.template_hook('event-vc-room-list-item-labels', self._render_vc_room_labels)
+        self.inject_bundle('main.js', WPSimpleEventDisplay)
+        self.inject_bundle('main.js', WPVCEventPage)
+        self.inject_bundle('main.js', WPVCManageEvent)
+
+    @property
+    def default_settings(self):
+        return dict(VCPluginMixin.default_settings, **{
+            'assistant_id': config.SUPPORT_EMAIL,
```

Comment:
hmm not sure about using a default here... seems safer to really force the admin to enter something instead of defaulting to something that may or may not be the correct email, especially since they need to enter the tokens etc anyway

Reply:
(when ditching this default the settings can go back to a class-level `default_settings = {...}` attribute again)

Reply:
Good point.